opts/envdir.go

```diff
@@ -0,0 +1,43 @@
+package opts
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"strings"
+)
+
+/*
+Read in a directory with files acting as environment variables.
+*/
+func ParseEnvDir(directory string) ([]string, error) {
+	dir, err := ioutil.ReadDir(directory)
+	if err != nil {
+		return []string{}, err
+	}
+
+	lines := []string{}
+	for _, info := range dir {
+		// ignore directories
+		if !info.IsDir() {
+			variable := info.Name()
+			if strings.ContainsAny(variable, whiteSpaces) {
```

Comment:
Given that this directly uses the filename as the name of the env-var, I can think of other file names to excluded as well (hidden files?) also, only names that are valid for environment variables?

A quick search for what characters are allowed; http://stackoverflow.com/questions/2821043/allowed-characters-in-linux-environment-variable-names